### PR TITLE
Docs ia

### PR DESCRIPTION
**Pull request by nicoleslaw**
For #62

Two questions:

* Not sure the CSS is styling correctly under /theming, please recheck.
* Is the release policy okay under /contributing, or should it be under /technology?

### CODE REVIEW

File: _pages/story.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 layout: default
-title: History, present, and future
+title: History and press
 ---
 
-# History, present, and future
+# History and press
 
-We would like other agencies, developers, and groups to use eRegulations and adapt it. Here's some insight into the current status of the project and where it's going.
+The CFPB released its initial version of eRegulations [in 2013](http://www.consumerfinance.gov/blog/making-regulations-easier-to-use/), and 18F and ATF began reusing it for ATF [in 2015](https://18f.gsa.gov/2015/12/09/an-open-source-government-is-a-faster-more-efficient-government/).
```
Comment (cmc333333) [inline _pages/story.md]: The language was broken before, but looks like some oddness around "18F and ATF began reusing it for ATF"

File: _pages/theming.md [hunk 2]
```diff
@@ -1,104 +1,100 @@
...
-You can do most font theming by including the basic font mixins to create specific formatting rules for different types of content. Here’s an example that declares a basic font and then creates a variation with a particular size and line height to use for a specific type of content within the theme:
+You can do most font theming by including the basic font mixins to create specific formatting rules for different types of content. Here’s an example that declares a basic font and then creates a variation with a particular size and line height for a specific type of content within the theme:
 
-``` scss
-@mixin sans-font-regular {
-  font-family: "Source Sans Pro", Arial, sans-serif;
-  font-weight: 400;
-  font-style: normal;
-}
+``` scss
```
Comment (cmc333333) [inline _pages/theming.md]: Looks like these code blocks aren't displaying with syntax highlighting for me. I may have some local hang up, but wanted to check with you.

**Review by cmc333333 — APPROVED** [review]
I think this re-org and cleanup has been long overdue, thanks @nicoleslaw!

Just a few comments, but we can merge as is.

File: _pages/story.md (hunk 1 above)
Comment (nicoleslaw) [reply _pages/story.md]: Ah yeh, I noticed that but wasn't sure if I should edit it. I simplified it a bit.

File: _pages/theming.md (hunk 2 above)
Comment (nicoleslaw) [reply _pages/theming.md]: I am having the same issue, but not sure why it's not working. I repasted the code from the live site to fix the spacing. Do you want to go ahead and merge this, and assume it's a local jekyll or gulp issue?

**Pull request merged**